packages/client/src/components/app/forms/FieldGroup.svelte

```diff
@@ -13,7 +15,18 @@
     class="spectrum-Form"
     class:spectrum-Form--labelsAbove={labelPosition === "above"}
   >
-    <slot />
+    <Section {type}>
```

Comment:
Could we instead check first if labels are left or above, and then only render the section if we're using above? That way we can continue to just render a `slot` of left labels, allowing the table layout to work without another form node.

Reply:
That's nicer - thanks!